manifests/kustomize/env/cert-manager/platform-agnostic-multi-user-k8s-native/kustomization.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../platform-agnostic-k8s-native
+  - ../../../third-party/metacontroller/base
+  - ../../../base/installs/multi-user
+  - ../../../base/metadata/options/istio
+  - ../../../third-party/argo/installs/cluster
+  - ../../../third-party/mysql/options/istio
+  - ../../../third-party/minio/options/istio
```

Does this work instead since you are reincluding the patches from `platform-agnostic-k8s-native` anyways?
```suggestion
  - ../platform-agnostic-multi-user
```